src/pcapi/core/search/__init__.py

```diff
@@ -53,6 +53,12 @@ def async_index_venue_ids(venue_ids: Iterable[int]) -> None:
     backends = _get_backends()
     for backend in backends:
         try:
+            # Rebuild venues in order to filter them.
+            # Keeping venue_ids as input parameter makes the whole
+            # indexing api clearer: all async_xyz function take ids as
+            # inputs.
+            venues = Venue.query.filter(Venue.id.in_(venue_ids))
+            venue_ids = [venue.id for venue in venues if venue.isPermanent]
```

Comment:
On pourrait laisser PostgreSQL filtrer. Ce sera sûrement plus performant que si le filtre est fait côté Python.

```suggestion
            venue_ids = {venue_id for venue_id, in Venue.query.filter(Venue.id.in_(venue_ids), Venue.isPermanent.is_(True)).with_entities(Venue.id)}
```

Reply:
oula oui tu as raison.